databricks/koalas/internal.py

```diff
@@ -793,12 +793,26 @@ def to_pandas_frame(self) -> pd.DataFrame:
             ]
         return pdf
 
+    @lazy_property
+    def applied(self):
+        """ Copy the immutable InternalFrame with the updates applied. """
+        sdf = self.spark_frame.select(self.spark_columns + list(HIDDEN_COLUMNS))
+        if self.spark_column is None:
+            return self.copy(
+                spark_frame=sdf,
+                data_spark_columns=[scol_for(sdf, col) for col in self.data_spark_column_names],
+            )
+        else:
```

Comment:
Just for double checking.
`if` for returning the applied sdf for the `DataFrame` and `else` for the `Series`, right ?

Reply:
Yes, right.

Reply:
Thanks !